helpers/tests/s3.rs

```diff
@@ -0,0 +1,96 @@
+#![cfg(feature = "s3")]
```

Comment:
I think we should find a way to maintain the tests, at least, in case this PR added a regression.